modules/javafx.graphics/src/main/java/com/sun/javafx/tk/quantum/UploadingPainter.java

```diff
@@ -38,7 +38,7 @@
  * UploadingPainter is used when we need to render into an offscreen buffer.
  * The PresentingPainter is used when we are rendering to the main screen.
  */
-final class UploadingPainter extends ViewPainter implements Runnable {
+final class UploadingPainter extends ViewPainter {
```

Comment:
I'm not sure I see the value in making this change.

Reply:
Since this is an internal class (not public API), I won't object to this change. I do think it's a rather pointless change, but I'll leave it up to you.

Reply:
In this particular case, it doesn't matter (especially since it's not public API), so I'm OK with the change. Where I _do_ think being more explicit is useful, is when a subclass `S` of a base class `B` wants to indicate that it has the property of being an instance of an interface `I` independent of whether or not its superclass `B` happens to implement `I`.